### PR TITLE
feat: add Flex shim and flex item mixins

### PR DESCRIPTION
**Pull request by chpalac**
## Overview

Add `Flex` shim and `FlexItem` mixin to v0-v9 migration package.

Screenshot 2023-02-27 161458

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/018e48731c7511dc205d74498b4c075624d2e01f'>018e48731c7511dc205d74498b4c075624d2e01f</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-migration-v0-v9/package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "@griffel/react": "^1.5.2",
     "tslib": "^2.1.0",
     "@fluentui/react-components": "^9.15.6",
-    "@fluentui/react-northstar": "^0.66.2"
+    "@fluentui/react-northstar": "^0.66.2",
+    "lodash": "^4.17.15"
```
Comment (ling1726) [inline packages/react-components/react-migration-v0-v9/package.json:43]:
```suggestion
```

We can use modern JS now, let's not add lodash 😬😬😬

**Review by ling1726 — CHANGES_REQUESTED** [review]
Let's remove the lodash dependency plz

File: packages/react-components/react-migration-v0-v9/package.json (hunk 1 above)
Comment (chpalac) [reply packages/react-components/react-migration-v0-v9/package.json]: Removed

**Pull request merged**